source/dotnet/Measurements.Abstractions/Api/Models/MeasurementPointDto.cs

```diff
@@ -8,5 +8,13 @@
 /// <param name="Quality">Quality of measurement.</param>
 /// <param name="Unit">Unit of measurement.</param>
 /// <param name="Resolution">Resolution of measurement.</param>
-/// <param name="Created">Timestamp defining when this point was created.</param>
-public sealed record MeasurementPointDto(int Order, decimal Quantity, Quality Quality, Unit Unit, Resolution Resolution, DateTimeOffset Created);
+/// <param name="CreatedInDataHub">Timestamp defining when this point was created in DataHub.</param>
+/// <param name="CreatedByGridAccessProvider">Timestamp defining when this point was created by Grid Access Provider.</param>
+public sealed record MeasurementPointDto(
+    int Order,
+    decimal Quantity,
+    Quality Quality,
+    Unit Unit,
+    Resolution Resolution,
+    DateTimeOffset CreatedInDataHub,
```

Comment:
If CreatedInDataHub corresponds to `created` in Gold, I would prefer a naming like

```suggestion
    DateTimeOffset PersistedInDataHub,
```

Reply:
@mimse @dzhavat Will these renamings be okay with you?

Reply:
done